README.md

```diff
@@ -48,6 +51,26 @@ To run the application in `development` mode run:
 npm run dev
 ```
 
+### Aws localstack
+
+#### Install and run LocalStack AWS
+
+- Install [LocalStack AWS CLI](https://github.com/localstack/awscli-local)
+- Run AWS LocalStack Docker container:
+
+```bash
+docker run --pull=always -d -p 4566:4566 -p 4510-4559:4510-4559 localstack/localstack:latest
+```
```

Comment:
I have run both, but just assumed people will run the one in cdp-local-environment. But I guess for generic tenant development using the raw localstack is best to show as an example.